tests/Testcontainers.Tests/Unit/Configurations/WaitUntilMessageIsLoggedTest.cs

```diff
@@ -24,14 +24,15 @@ public WaitUntilMessageIsLoggedTest()
         .Build();
     }
 
-    public Task InitializeAsync()
+    public async ValueTask InitializeAsync()
     {
-      return _container.StartAsync(_cts.Token);
+      await _container.StartAsync(_cts.Token);
     }
 
-    public Task DisposeAsync()
+    public async ValueTask DisposeAsync()
     {
-      return _container.StartAsync();
+      // TODO: Should this really be StartAsync?
+      await _container.StartAsync();
```

Comment:
This really feels like it should call `StopAsync` or `DisposeAsync`

Reply:
👍 Yes, this is a mistake.

Reply:
Changed it to `DisposeAsync`

Reply:
Thanks